BUILD.bazel

```diff
@@ -240,6 +240,7 @@ container_bundle(
         "$(container_prefix)/$(image_prefix)cirros-custom-container-disk-demo:$(container_tag)": "//containerimages:cirros-custom-container-disk-image",
         "$(container_prefix)/$(image_prefix)fedora-cloud-container-disk-demo:$(container_tag)": "//containerimages:fedora-cloud-container-disk-image",
         "$(container_prefix)/$(image_prefix)fedora30-cloud-container-disk-demo:$(container_tag)": "//containerimages:fedora30-cloud-container-disk-image",
+        "$(container_prefix)/$(image_prefix)fedora32-cloud-container-disk-demo:$(container_tag)": "//containerimages:fedora32-cloud-container-disk-image",
```

Comment:
Can you please drop fedora30 and fedora32 and instead just bump fedora to 32? We should not add more and more images, instead we should bump and ensure that all tests work with the new version.

Reply:
Done.

Fingers crossed; I sincerely don't know how "traumatic" this bump will be to the existent tests. 

Let's see what CI makes of it.

Reply:
Let's hope, not too traumatic. But we have to go through these things in order to keep things usable and maintainable. Luckily we can adjust all that in one PR and see the outcome.

Reply:
@rmohr alright, seems ci didn't complain at all about the fedora bump.

I think this means the PR can take a 'serious' review.